.quickstart/quickstart.yml

```diff
@@ -13,4 +13,13 @@ destination_configurations:
   databricks:
     dispatch:
       - macro_namespace: dbt_utils
-        search_order: [ 'spark_utils', 'dbt_utils' ]
\ No newline at end of file
+        search_order: [ 'spark_utils', 'dbt_utils' ]
+public_models: [
+  "servicenow__user_enhanced",
```

Comment:
I don't know if this type of data would work. I've tested it against data that looks like it's on the Slab page.
It should look like this:
```
public_models:
  servicenow__user_enhanced
  servicenow__change_request_enhanced
  servicenow__user_aggregated
  servicenow__problem_enhanced
  servicenow__activity_summary
  servicenow__task_enhanced
  servicenow__incident_enhanced
```

Reply:
Hey @fivetran-markoteodorovic we modeled these public_models sections off of @FedorSafronov [PR here](https://github.com/fivetran/dbt_ad_reporting/pull/114). 

My understanding is that these should be in a list format. The suggested yml format doesn't represent a list, but the suggested changes in this PR do as well as the following format:
```yml
public_models:
  - servicenow__user_enhanced
  - servicenow__change_request_enhanced
  - servicenow__user_aggregated
  - servicenow__problem_enhanced
  - servicenow__activity_summary
  - servicenow__task_enhanced
  - servicenow__incident_enhanced
```

Reply:
@fivetran-markoteodorovic I'm sure that a "bulleted" list as @fivetran-joemarkiewicz wrote it will produce an array and also a json-like notation (`property: ['item1', 'item2']` will also produce an array). However, I'm not sure about the notation you mentioned. It may produce a list as well, but I'm not sure.

Reply:
Yes, maybe all of theme could be parsed. If Fedors changes passed tests then this will too.